src/Settings/Identity/Index.jsx

```diff
@@ -30,6 +34,12 @@ const Icon = styled.i`
 
 const bannerToggle = useCallback(() => setShowBanner(!showBanner), [showBanner]);
 
+useEffect(() => {
+  if (!props.idosConnected && props.connectIdOS) {
+    props.connectIdOS()
+  }
+}, [props.idosConnected]);
```

Comment:
Looks like we might have forgot to add `props.connectIdOS` to the dependencies array?

Reply:
Thanks for catch!